src/db/tables/profiles.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import initKnex from 'knex'
+import knexConfig from '../knexfile.js'
+
+import { ProfileData } from "../../external/onerep.js";
+
+const knex = initKnex(knexConfig)
+
+/**
+ * @param {number} onerepProfileId
```

Comment:
Doesn't this result in an object with a whole bunch of properties, rather than just the profile ID? (In fact, I think TypeScript should be able to infer the correct types if you add a type definition to `knex-tables.d.ts`.)

Reply:
Hm well this is `getProfileDetails`, that just takes the ID... the function below takes a `ProfileData` object so that seems fine?

Reply:
Oh whoops, I read `@returns` instead of `@param`. Never mind then! :sweat_smile: